x-pack/solutions/security/plugins/lists/server/saved_objects/migrations.ts

```diff
@@ -72,7 +74,7 @@ export const migrations = {
   '7.12.0': (
     doc: SavedObjectUnsanitizedDoc<ExceptionListSoSchema>
   ): SavedObjectSanitizedDoc<ExceptionListSoSchema> => {
-    if (doc.attributes.list_id === ENDPOINT_TRUSTED_APPS_LIST_ID) {
+    if (doc.attributes.list_id === ENDPOINT_ARTIFACT_LISTS.trustedApps.id) {
```

Comment:
Oh, so you are using it already, `ENDPOINT_ARTIFACT_LISTS` instead of `ENDPOINT_TRUSTED_APPS_LIST_ID`  ;)